unit_tests/child_chain/test_child_chain.py

```diff
@@ -167,6 +189,11 @@ def test_get_block(self, child_chain):
         expected = rlp.encode(child_chain.db.get_block(DUMMY_BLK_NUM)).hex()
         assert expected == child_chain.get_block(DUMMY_BLK_NUM)
 
+    def test_get_block_with_current_block_number(self, child_chain):
```

Comment:
This is a missing branch forgot to cover in my previous PR. Sneak in with this PR :P .